dataprofiler/labelers/data_processing.py

```diff
@@ -37,7 +37,7 @@ def __new__(
 class BaseDataProcessor(metaclass=abc.ABCMeta):
     """Abstract Data processing class."""
 
-    processor_type: str = None  # type: ignore[assignment]
+    processor_type: str = "baseprocessor"  # override this
```

Comment:
Hmm, will this causes issues in tests or validation to make sure this is set? Will a string give the false impression of naming?

what if we just do without assignment or does that fail?
`processor_type: str`

Reply:
This works internally but causes a mypy error due to the property being accessed by a unit test (says property doesn't exist). I'll look into whether that test can be modified

Reply:
Did a quick 2 sec check, might be related to: https://github.com/python/mypy/issues/4426